Automattic/jetpack
# Boost: Refactor module status to use option store

Fixes #29088

## Proposed changes:
* Use options-store for module status in PHP
* Added a backward compatibility module to transfer module status to option-store

Switching the JS side is coming in a separate PR.

### Other information:


## Jetpack product discussion


None

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Install boost and try toggling all modules on and off
* Make sure the functionality of each module continues to work
* Install boost using [current trunk](https://jurassic.ninja/create/?jetpack-beta&branches.jetpack-boost=master&wp-debug-log), toggle module statuses there and update to this feature branch.
* Make sure your old state transferred smoothly.
